templates/app/views/spree/products/_product_form.html.erb

```diff
@@ -15,7 +15,7 @@
     </div>
   <% end %>
   <%= render 'product_variants' %>
-  <% if @product.price and !@product.price.nil? %>
+  <% if @product.price %>
```

Comment:
~On the next line we are using `product` instead of `@product`, what about using the same variable here?~

Saw you did it in the next commit 💌